docs/docs/configuration/server.rst

```diff
@@ -229,6 +229,50 @@ timeout
     specify a timeout in seconds for blocking operations like the
     connection attempt.
 
+Mail
+----
+
+Configure the format the mail notification uses in the SMTP section.
+
+.. code-block:: ini
+
+    [mail]
+    subject_admin = {title}
+    subject_user = Re: New comment posted on {title}
+
+subject_admin
+    specify the subject format of the notification email sent to the admin.
+    Default: ``{title}``. 
+    Available variables:
+
+    {title}
+        The title of the isso thread.
+
+    {replier}
+        The name of the author of the comment. If the author has no name,
+        it will fallback to the term "Anonymous".
+
+subject_user
+    specify the subject format of the notification email sent to the subscribed
+    commenter. If it is set to two values, then the first one will be used when
+    ``{repliee}``'s comment is not the ``{receiver}``'s original one, the second one
+    will be used otherwise.
+    Default: ``Re: New comment posted on {title}``.
+    Available variables:
+
+    {title}
+        The title of the isso thread.
+
+    {replier}
+        The name of the author of the comment. If the author has no name,
+        it will fallback to the term "Anonymous".
+
+    {repliee}
+        The author whose comment is replied to.  If the author has no
+        name, it will fallback to the term "Anonymous".
+
+    {receiver}
+        The receiver of the email.
```

Comment:
Their name, email?

Reply:
Or you want me to s/The receiver/The name of the recipient ?

Reply:
If that's what the variable gets set to, yes.

Reply:
Okay, I'll change it.